package.json

```diff
@@ -18,19 +18,65 @@
+                "doxdocgen.generic.firstLine": {
+                    "description": "The first line of the comment that gets generated. If empty it won't get generated at all.",
+                    "type": "string",
+                    "default": "/**"
+                },
+                "doxdocgen.generic.commentPrefix": {
+                    "description": "The pre fix that is used for each comment line.",
+                    "type": "string",
+                    "default": " * "
+                },
+                "doxdocgen.generic.lastLine": {
+                    "description": "The last line of the comment that gets generated. If empty it won't get generated at all.",
+                    "type": "string",
+                    "default": "**/"
+                },
+                "doxdocgen.generic.newLineAfterBrief": {
+                    "description": "Whether to insert a newline after a brief.",
                     "type": "boolean",
                     "default": true
+                },
+                "doxdocgen.generic.newLineAfterParams": {
+                    "description": "Whether to insert a newline after the params.",
+                    "type": "boolean",
+                    "default": false
+                },
+                "doxdocgen.generic.newLineAfterTParams": {
+                    "description": "Whether to insert a newline after the template params.",
+                    "type": "boolean",
+                    "default": false
+                },
+                "doxdocgen.generic.includeTypeAtReturn": {
+                    "description": "Whether include type information at return.",
+                    "type": "boolean",
+                    "default": false
+                },
+                "doxdocgen.generic.briefTemplate": {
+                    "description": "The template of the brief DoxyGen line that is generated. If empty it won't get generated at all.",
+                    "type": "string",
+                    "default": "@brief "
+                },
+                "doxdocgen.generic.paramTemplate": {
+                    "description": "The template of the param DoxyGen line(s) that are generated. If empty it won't get generated at all.",
+                    "type": "string",
+                    "default": "@param {param} "
+                },
+                "doxdocgen.generic.tparamTemplate": {
+                    "description": "The template of the template parameter DoxyGen line(s) that are generated. If empty it won't get generated at all.",
+                    "type": "string",
+                    "default": "@tparam {param} "
+                },
+                "doxdocgen.generic.returnTemplate": {
+                    "description": "The template of the return DoxyGen line that is generated. If empty it won't get generated at all.",
+                    "type": "string",
+                    "default": "@return {param} "
```

Comment:
The `{param}` here is a bit confusing I think.
Maybe choose `{type}` for this?

Reply:
Yeah that might be better. I just used the same string everywhere to make the code more clean. I can add a different name here.

Reply:
Fixed